src/core/SkipToContent/SkipToContentLink.tsx

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/skip-to-content.css';
+
+export type SkipToContentLinkProps = {
+  /**
+   * Localize 'Skip to main content' label if needed.
+   * @default 'Skip to main content'
+   */
+  children?: React.ReactNode;
+} & Omit<CommonProps, 'href'>;
+
+/**
+ * Link is for screen reader and keyboard users and will not be visible unless tabbed to.
+ * Provides a shortcut to the main content of the page without navigating through the header, etc.
+ * Should be placed just inside the body. Used in conjunction with `SkipToContentMarker`.
+ * @example
+ * <body><SkipToContentLink /> ... </body>
+ * <body><SkipToContentLink>{localizedLabel}</SkipToContentLink> ... </body>
+ */
+export const SkipToContentLink = (props: SkipToContentLinkProps) => {
+  const { children = 'Skip to main content', className, ...rest } = props;
```

Comment:
While it's nice that the user doesn't have to specify children, I am not so sure.

Do we really want to set default text here? The "Skip to" part is fine, but the "main content" part will probably depend on the app's context, e.g. "Skip to search results", "Skip to blog post" or even something like "Skip iframe" 

Another concern is that the user might forget to localize this.

Reply:
Default text is set in other components, such as `IdeasButton`, `Footer`, and `DatePicker`. Also, this phrase "Skip to content" or "Skip to main content" is standard for this feature and is found on large sites (Google, Amazon, LinkedIn, etc.) without specialized text.

Reply:
> this phrase "Skip to content" or "Skip to main content" is standard for this feature 

Your own example provides two variations, which means it's not really standard.

Also don't discard my concern about multiple links which need different text.

```html
<body>
  <SkipToContentLink href="#main">Skip to content</SkipToContentLink>
  <header>Main page header with some links to other parts of site...</header>

  <main id="main">
    <nav style="position: relative">
      <SkipToContentLink href="#table-skip" style="position: absolute">Skip table of contents</SkipToContentLink>
      <ul>Table of contents for this page...</ul> 
    </nav>

    <p id="table-skip">Content before the iframe…</p>

    <div style="position: relative">
      <SkipToContentLink href="#iframe-skip" style="position: absolute">Skip iframe content<SkipToContentLink>
      <iframe src="embed-url-from-other-site"></iframe>
    </div>

    <p id="iframe-skip">Content after the iframe…</p>
  </main>
</body>
```

Reply:
Ok, fair. If you are okay with having this inconsistent usage (`<SkipToContent />` vs `<SkipToContent>Skip table of contents</SkipToContent>` in the same page), then it's fine.

I will leave this comment open for visibility.